rpcs3/Emu/Io/Dimensions.cpp

```diff
@@ -384,8 +384,8 @@ u32 dimensions_toypad::load_figure(const std::array<u8, 0x2D * 0x04>& buf, fs::f
 	std::memcpy(figure.data.data(), buf.data(), buf.size());
 	// When a figure is added to the toypad, respond to the game with the pad they were added to, their index,
 	// the direction (0x00 in byte 6 for added) and their UID
-	std::array<u8, 32> figure_change_response = {0x56, 0x0b, figure.pad, 0x00, figure.index, 0x00};
-	std::memcpy(&figure_change_response[6], buf.data(), 7);
+	std::array<u8, 32> figure_change_response = {0x56, 0x0b, figure.pad, 0x00, figure.index, 0x00,
+		buf[0], buf[1], buf[2], buf[4], buf[5], buf[6], buf[7]};
```

Comment:
This seems like a typo
```suggestion
		buf[0], buf[1], buf[2], buf[3], buf[4], buf[5], buf[6]};
```

Reply:
same as below